x/ecocredit/server/marketplace/query_sell_orders.go

```diff
@@ -0,0 +1,116 @@
+package marketplace
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/orm/model/ormlist"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	marketplacev1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/marketplace/v1"
+	"github.com/regen-network/regen-ledger/types/ormutil"
+	v1 "github.com/regen-network/regen-ledger/x/ecocredit/marketplace"
```

Comment:
`api` and `marketplace`? Would be easier to maintain and no need to specify `v1`.